pkg/controller/daemon/daemoncontroller.go

```diff
@@ -37,16 +36,16 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/framework"
 	"k8s.io/kubernetes/pkg/controller/framework/informers"
+	"k8s.io/kubernetes/pkg/kubelet/util/format"
```

Comment:
it's weird to be depending on a pkg/kubelet/util package in the controller manager. Perhaps we should move required functionality to pkg/util